locales/es.js

```diff
@@ -0,0 +1,155 @@
+export const es = {
+  welcome: {
+    title: 'Abra su nueva cuenta bancaria',
+    description_p_1: 'Para abrir una cuenta bancaria, necesitaremos verificar su identidad.',
+    description_p_2: 'Solo tomará un par de minutos.',
+    next_button: 'Verificar identidad',
+  },
+  document_selector: {
+    title: 'Verifique su identidad',
+    hint: 'Seleccione el tipo de documento que desea cargar',
+  },
+  capture: {
+    driving_licence: {
+      front: {
+        title: 'Frente de la licencia',
+        instructions: 'Cargue el frente de la licencia desde su computadora',
```

Comment:
Again, computadora is more a south american term. Ordenador is the Spanish term. What audience are we aiming for?

Reply:
We are aiming to a South American/Mexican audience.